src/consensus/raft/test/enclave.cpp

```diff
@@ -44,22 +44,22 @@ TEST_CASE("Enclave put")
 
 TEST_CASE("Enclave record")
 {
-  ringbuffer::Circuit eio_leader(1024);
-  std::unique_ptr<WFactory> writer_factory_leader =
-    std::make_unique<WFactory>(eio_leader);
+  ringbuffer::Circuit eio_primary(1024);
```

Comment:
These _could_ stay leader/follower too, since they're Raft-specific tests, but I'm 50/50 on this.

Reply:
Changed back since raft.h now refers to leader/follower